### ISSUE TITLE
request-capture-har is breaking yarn on node 4

### ISSUE THREAD

**Issue by binduwavell** [yarnpkg/yarn#2996]
The dep in package.json is ^1.1.4, so you all bring in 1.2.1, which is broken on node 4.

**Comment by Daniel15** [comment]
For what it's worth, this is one reason we suggest that people install Yarn using our Windows installer, Debian package, or Homebrew. request-capture-har is locked to 1.1.4 in yarn.lock, so our release will **always** contain 1.1.4 until someone changes the version in yarn.lock: https://github.com/yarnpkg/yarn/blob/6bb3cf5c80d400b26ddb90d4b39b8d0fd9021406/yarn.lock#L3862-L3864. npm shrinkwrap doesn't fully solve the issues, so there's no way of us getting a fully repeatable/deterministic installation via npm 😕

**Comment by binduwavell** [comment]
Hi Daniel, I completely understand. In this case I'm using yarn in a .travis.yml file and all I could find was folks doing `npm install -g yarn` in this file to get a recent build of yarn (I have a java machine but need yarn as well.)

I've subsequently read through a bunch of the install docs and come up with this, which actually works way better than trying to add via the apt repo (or npm):

```yaml
install:
  - nvm install $NODE_VERSION
  - wget https://yarnpkg.com/install.sh
  - chmod 755 install.sh
  - ./install.sh
  - $HOME/.yarn/bin/yarn
```

**Comment by Daniel15** [comment]
@binduwavell  - Yeah, the installation script is a good idea. You can lock down the version of Yarn you're using if you like (eg. `./install.sh --version 0.21.3`). Using the Debian repo should work fine too.

This is documented on the Yarn site too: https://yarnpkg.com/lang/en/docs/install-ci/#travis-tab

**Comment by paulirish** [comment]
I've fixed the bug, shipping request-capture-har@1.2.2 which should sort out your builds.
Apologies for the breakage, folks.

**Comment by binduwavell** [comment]
Thanks Paul
